best-practices/README.md

```diff
@@ -0,0 +1,71 @@
+Best Practices
+=============
+
+The following are locally agreed upon best practices. As often as possible,
+these will inherit from practices within the larger open source development
+community at large. These best practices may, and likely will, change in the
+future. But that change will be incorporated via group consensus and documented
+here.
+
+* [Documentation](documentation.md)
+* [Project README's](project_readme.md)
+* [Deployment](deployment.md)
+
+
+General
+-------
+
+* These are not to be blindly followed; strive to understand these and ask
+  when in doubt.
+* Don't duplicate the functionality of a built-in library.
+* Don't swallow exceptions or "fail silently."
```

Comment:
The one case I can remember was doing XML parsing of MARCXML that had been generated from binary MARC records.  I was doing stream parsing to handle very large XML files, and had to write an error handler, which essentially did nothing. There was extensive validation of the data before the XML parsing step, so I remember feeling like it was very cumbersome to just ignore the (non-existent) error messages.

Anyway, probably a corner case that doesn't have to be reflected in the top-level general guidelines.